addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/icons/Apple.svg

```diff
@@ -1,23 +1,23 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>

```

Comment:
Line ending changes? (The other new SVG files could also use \n.)

Reply:
Hmm I’ll check

Reply:
I've checked about a half dozen, I can fix them manually (outside of eclipse). But of those that I checked they were actually CRLF and are now LF.

Let me know what I should do.

Reply:
Ok I think I got the "changes" related to EOL sorted. (For this PR)